IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_TrackFacilityPart/DocPropertySet.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_TrackFacilityPart" UniqueId="011923c1-fbf2-84f9-6215-e3ea43caebc9" ApplicableType="IfcFacilityPart" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="IsBallastTrack_3GWQlZqZbCOhyZObZZzqs" />
+		<DocProperty xsi:nil="true" href="IsCWR_1mrqmXkov0pfRjK0zoCDrB" />
+		<DocProperty xsi:nil="true" href="IsSunExposed_3jUxqj7iX3zerVO_VEXgG7" />
+		<DocProperty xsi:nil="true" href="DetailedDrawingID_0v98BA4e53VwGZS60uKm8C" />
+		<DocProperty xsi:nil="true" href="TrackPartType_3smcaufEP68Po49oZGOONY" />
+		<DocProperty xsi:nil="true" href="TypeOfTrackSuportingStructure_0HyLCL3Pz3kwIHGXDrwFlL" />
```

Comment:
From what i can tell the content of this property is covered by spatial interference relationships which would link actual instances instead of just typing of the supporting structure

Reply:
I still keep this one, for the reasons that 1) it is not fully tested regarding spatial structure for this scenario and 2) in railway project, it may not have spatial structure from other domains.

Reply:
fair, some of the enumerations are not interference related